### ISSUE TITLE
The CottonFox Table has some issues showing the Balls.

### ISSUE THREAD

**Issue by BluWizard10** [VRCBilliards/vrcbce#134]
The 1 and Cue Ball seem to be missing on the Esnya UI prefab for the CottonFox Table. It usually happens right at the start of the game.

Since the video is over 10MB, [here is a link to the video](https://imgur.com/a/ZTh17bg)

**Comment by BluWizard10** [comment]
UPDATE: Since I'm good friends with Juice, I asked him about it. We tried a few options, such as turning off Dynamic Occlusion on the Balls and it still didn't seem to solve the problem. So, it's gotta do with the Code of the Table somehow.

Again, this is on the latest 1.5.1 version of the prefab.

**Comment by Metamaniac** [comment]
Wasn't able to repro this in a clean unity install, was this a clean install?

**Comment by BluWizard10** [comment]
> Wasn't able to repro this in a clean unity install, was this a clean install?

Nope, this was in an existing project. I was upgrading from 1.5.0 to 1.5.1, following the instructions required (Delete the VRCBCE folder prior to importing the new version).

This comment also applies to Issue #135.

**Comment by Metamaniac** [comment]
does the Meshes section of the Core Table Code inspector look like this?
![image](https://user-images.githubusercontent.com/6299186/147146641-1ed9685b-e816-4533-ac91-beea406c8260.png)
and are there any other changes that would cause Unity to show the UdonBehaviour listed being different? (try clicking the UdonBehaviour entry and hitting "revert" at the top of the popup)
![image](https://user-images.githubusercontent.com/6299186/147147070-60d36468-a20b-42d7-b3a6-8cf0aeb5a2f9.png)

**Comment by BluWizard10** [comment]
I'm unsure! I had to revert back to 1.5.0 and it worked again. I tried following those steps and making sure those things were correct in a fresh Project and it worked perfectly.

I tried a fresh Scene too in the project where I'm having those issues, and still got it. But in a whole fresh Project, it works fine...
